src/modules/nmx/geometry/Config.h

```diff
@@ -42,12 +42,12 @@ class Config : public VMM3Config {
     uint16_t SizeX{1280};
     uint16_t SizeY{1280};
     uint16_t NumPanels{4};
-    uint16_t MaxXSpan{10};
-    uint16_t MaxYSpan{10};
-    uint16_t MinXSpan{1};
-    uint16_t MinYSpan{1};
-    uint16_t MaxXGap{10};
-    uint16_t MaxYGap{10};
+    uint16_t MaxSpanX{10};
+    uint16_t MaxSpanY{10};
+    uint16_t MinSpanX{1}; // should be larger?
```

Comment:
Shall we use ToDo to mark these points for a later review? We can time to time roll over the code search for ToDo's and then create an task/issue from them. But hinting our uncertainty in the code without make this point traceable I see no reason. But let me know if you have different experience on this.

Reply:
Yes we should and in general I do that using /// \todo - I'll add it here too